.env

```diff
@@ -0,0 +1,3 @@
+CLOUDINARY_CLOUD_NAME =hanhngo
+CLOUDINARY_API_KEY =914389718193435
+CLOUDINARY_API_SECRET =6Nrz2S6inrGLN4W1fOMbCQghjUc
```

Comment:
this file shouldn't be here hanh! lets get rid of it. copy the contents somewhere else temporary and do:
```
git rm .env
```
the add the file to `.gitignore`and create it again and put the stuff back in